osu.Game/Overlays/Music/MusicKeyBindingHandler.cs

```diff
@@ -74,7 +74,7 @@ public bool OnPressed(KeyBindingPressEvent<GlobalAction> e)
                                 break;
 
                             case PreviousTrackResult.Previous:
-                                onScreenDisplay?.Display(new MusicActionToast(GlobalActionKeyBindingStrings.MusicPrev, e.Action));
+                                onScreenDisplay?.Display(new MusicActionToast("Test", e.Action));
```

Comment:
Surely accidental / unintended change?

Reply:
Yeah, sorry, was trying to figure out why some strings can't be localized accidentally, will fix